src/components/SideBar.jsx

```diff
@@ -0,0 +1,113 @@
+      </div>
+
+      {/* Middle main section */}
+
+      <div className="flex-1 overflow-y-auto mt-4 space-y-6">
+        <div className="space-y-2">
+          <div className="flex items-center gap-3 px-4 py-1 h-7 cursor-pointer hover:bg-muted-foreground">
+            <Trello className="h-4 w-4" />
+            <h4 className="text-sm flex-1">Boards</h4>
+          </div>
+
+          <div className="flex items-center gap-3 px-4 py-1 h-7 cursor-pointer hover:bg-muted-foreground">
+            <UserRound className="h-4 w-4" />
+            <h4 className="text-sm flex-1">Member</h4>
+            <Plus className="h-6 w-6 p-1 rounded-sm transition-all hover:bg-muted" />
+          </div>
+
+          <div className="flex items-center gap-3 px-4 py-1 h-7 cursor-pointer hover:bg-muted-foreground">
+            <Settings className="h-4 w-4" />
+            <h4 className="text-sm flex-1">Workspace Settings</h4>
+            <ChevronDown className="h-6 w-6 p-1 rounded-sm transition-all hover:bg-muted" />
+          </div>
+        </div>
+
+        <div className="space-y-2">
+          <h1 className="ml-4 font-bold text-sm pb-1">Workspace views</h1>
+
+          <div className="flex items-center gap-3 px-4 py-1 h-7 cursor-pointer hover:bg-muted-foreground group">
+            <Table className="h-4 w-4" />
+            <h4 className="text-sm flex-1 italic">Table</h4>
+            <Ellipsis className="h-6 w-6 p-1 rounded-sm transition-all hidden hover:bg-muted group-hover:inline-block" />
+          </div>
+
+          <div className="flex items-center gap-3 px-4 py-1 h-7 cursor-pointer hover:bg-muted-foreground group">
+            <CalendarDays className="h-4 w-4" />
+            <h4 className="text-sm flex-1 italic">Calendar</h4>
+            <Ellipsis className="h-6 w-6 p-1 rounded-sm transition-all hidden hover:bg-muted group-hover:inline-block" />
+          </div>
+        </div>
+
+        <div className="space-y-2">
+          <div className="flex items-center gap-3 px-4 py-1 h-7 cursor-pointer group">
+            <h4 className="text-sm flex-1 font-bold">Your boards</h4>
+            <Ellipsis className="h-6 w-6 p-1 rounded-sm transition-all hidden hover:bg-muted group-hover:inline-block" />
+            <Popover>
+              <PopoverTrigger>
+                <Plus className="h-6 w-6 p-1 rounded-sm transition-all hover:bg-muted" />
+              </PopoverTrigger>
+              <PopoverContent
+                align="start"
+                sideOffset={30}
+                className={cn("w-[22rem]")}
+              ></PopoverContent>
+            </Popover>
+          </div>
+
+          {/* list of baords here */}
+        </div>
+      </div>
+
+      {/* Footer Div */}
+
+      <div className="px-4 py-2">
+        <Button className={cn("w-full")}>Clear Data</Button>
+      </div>
+    </aside>
+  );
+};
+
+export default SideBar;
```

Comment:
@oto-wtag Bhai, why we are having this file changes in this PR? I assumed this PR should only have the changes related to project setup, no?

Reply:
The story originally included changes for the layout (including navigation), and while it was later updated to focus on just the project setup, I kept the layout changes in the code. I figured I’d need the navigation bars later anyway, so I didn’t remove them.